### ISSUE TITLE
[Feature Request] Add different settings of number of columns for the App Drawer and the Favourite box

### ISSUE THREAD

**Issue by kud** [MM2-0/Kvaesitso#507]
Hey,

It could be great to have the ability to define a different number of columbs for the App Drawer grid and the Favourite box grid.

It's what Lawchair does. 😌

Thanks!

**Comment by exstntlmsnthrp** [comment]
This was already added in a previous update. 

Settings > Grid & icons

![Screenshot_2.png](https://github.com/MM2-0/Kvaesitso/assets/57594290/951b690f-de52-4e72-ad8e-32c9e81e2181)

Please make sure you review all settings and review the help before making requests that already exist or have been requested by others. As well as read/search existing GitHub comments and bugs.

**Comment by MM2-0** [comment]
I think he means two *different* settings. Ateast that's how I understand it.

**Comment by kud** [comment]
Exactly. I would like to have 2 settings, one for favourite, one for app drawer.
